### ISSUE TITLE
`https://t.me/...` links dont get mapped to telegram icon

### ISSUE THREAD

**Issue by jessupjn** [couetilc/react-social-icons#287]
![Screenshot 2024-02-07 at 3 15 40 PM](https://github.com/couetilc/react-social-icons/assets/3454809/5cee595c-fba3-4928-be64-0c304d01eb75)

I'll come back to this when I can, but if anyone else has the time this should be a quick task. The telegram icons already look like they're installed. Just need to be mapped through...

**Comment by preston176** [comment]
![image](https://github.com/couetilc/react-social-icons/assets/49411983/78338515-9b4f-416f-bd99-96e39669c4c2)


https://t.me is working on my side

**Comment by couetilc** [comment]
@jessupjn please share the version of `react-social-icons` you are using and a minimal reproduction of the issue withcode.

**Comment by jessupjn** [comment]
I am on `react-social-icons@6.10.0`.

My usage is pretty simple. This doesn't work (and results in the generic 'share' icon):
```
<SocialIcon
    style={{ width: 32, height: 32 }}
    url={'https://t.me/jack_switchboard'}
    target={'_blank'}
    bgColor={'transparent'}
    fgColor={'#FFF'}
    />
```
When I swap the url for `https://telegram.me/jack_switchboard` - I see the telegram icon as I expect. This is how I've resolved my issue - but figured the shortened version should be better supported as well.

**Comment by couetilc** [comment]
@jessupjn fixed in version 6.11.0. Thank you for raising this issue and for using `react-social-icons`.
